boards/laird_connect/bl654_dvk_bl654pa/Kconfig.defconfig

```diff
@@ -0,0 +1,18 @@
+# BL654 DVK board configuration
+
```

Comment:
file is in wrong path and probably doesn't need updating in the original

Reply:
This was an accidental commit. I'll remove and rebase from collab-hwm